lib/gui-utils/src/GUIUtils.cpp

```diff
@@ -245,16 +262,74 @@ void GUIUtils::displaySensorAverage(int average) {
         displayBigLabel(0, " brown");
 #endif
     }
-#endif
     char output[4];
     sprintf(output, "%03d", average);
-    displayCenterBig(output);
+    displayCenterBig(output, deviceType);
+  }
+    else{                           //PM sensors
+              if (average < 600) {
+#ifdef TTGO_TQ
+        u8g2.drawXBM(1, 0, 32, 32, SmileFaceGood);
+        displayBigEmoticon("GOOD");
+        displayBigLabel(66, "/green");
+#else
+        u8g2.drawXBM(0, 1, 32, 32, SmileFaceGood);
+        displayBigEmoticon("  GOOD");
+        displayBigLabel(0, " green");
+#endif
+    } else if (average < 800) {
+#ifdef TTGO_TQ
+        u8g2.drawXBM(1, 0, 32, 32, SmileFaceModerate);
+        displayBigEmoticon("MODERATE");
+        displayBigLabel(90, "/yel");
+#else
+        u8g2.drawXBM(0, 1, 32, 32, SmileFaceModerate);
+        displayBigEmoticon("MODERATE");
+        displayBigLabel(0, "yellow");
+#endif
+    } else if (average < 1000) {
+#ifdef TTGO_TQ
+        u8g2.drawXBM(1, 0, 32, 32, SmileFaceUnhealthy);
+        displayBigEmoticon("UNHEALT");
+        displayBigLabel(84, "/red");  //OK
+#else
+        u8g2.drawXBM(0, 1, 32, 32, SmileFaceUnhealthy);
+        displayBigEmoticon("UNHEALT");
+        displayBigLabel(0, "  red");
+#endif
+    } else if (average < 1400) {
+#ifdef TTGO_TQ
+        u8g2.drawXBM(1, 0, 32, 32, SmileFaceVeryUnhealthy);
+        displayBigEmoticon("V UNHEA");
+        displayBigLabel(84, "/viol");  //OK
+#else
+        u8g2.drawXBM(0, 1, 32, 32, SmileFaceVeryUnhealthy);
+        displayBigEmoticon("V UNHEA");
+        displayBigLabel(0, "violet");
+#endif
+    } else {
+#ifdef TTGO_TQ
+        u8g2.drawXBM(1, 0, 32, 32, SmileFaceHazardous);
+        displayBigEmoticon("HAZARD");
+        displayBigLabel(78, "/brown");
+#else
+        u8g2.drawXBM(0, 1, 32, 32, SmileFaceHazardous);
+        displayBigEmoticon("HAZARD");
+        displayBigLabel(0, " brown");
+#endif
+    }
+    char output[4];
+    sprintf(output, "%04d", average);
```

Comment:
please reformat this block of code, I think that it don't changed, maybe is only that